src/app/components/Setting/index.tsx

```diff
@@ -2,17 +2,18 @@ type Props = {
   title: string;
   subtitle: string;
   children: React.ReactNode;
+  link?: string[];
 };
 
-function Setting({ title, subtitle, children }: Props) {
+function Setting({ title, subtitle, children, link }: Props) {
```

Comment:
I am unhappy about this. that' is the problem with such abstractions... you loose flexibility. 
If we now add custom code or attributes here and there it increases a lot of the complexity. 
No other Setting needs the link, so I would actually avoid adding this. is that link really that valuable?
and if? can we pass in html as the subtitle? (without removing escaping)

Reply:
`Trans` sounds interesting. 

I am also OK with removing the link

Reply:
I was able to remove the prop and use the `Trans` comp.